src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
```

Comment:
pls refactor your code so that you won't have to disable `no-nested-ternary` rule